mrmustard/math/backend_manager.py

```diff
@@ -888,6 +933,18 @@ def reshape(self, array: Tensor, shape: Sequence[int]) -> Tensor:
         """
         return self._apply("reshape", (array, shape))
 
+    def round(self, array: Tensor, decimals) -> Tensor:
```

Comment:
For consistency, add `decimals: int` here and remove the type hint from the "Args" section